VolunteerApp/ClientApp/src/app/signupBtn/signupBtn.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component } from '@angular/core';
+@Component({
+  selector: 'app-signup-button',
+  templateUrl: './signupBtn.component.html',
```

Comment:
Please rename according to best practices

https://medium.datadriveninvestor.com/best-practice-naming-conventions-in-angular-af26430d2464

Reply:
fixed